parsing/parsetree.mli

```diff
@@ -215,10 +215,12 @@ and pattern_desc =
 
            Invariant: n >= 2
         *)
-  | Ppat_construct of Longident.t loc * pattern option
-        (* C                None
-           C P              Some P
-           C (P1, ..., Pn)  Some (Ppat_tuple [P1; ...; Pn])
+  | Ppat_construct of
+      Longident.t loc * (pattern * string loc list) option
+        (* C                    None
```

Comment:
Nitpick : I would find more natural to follow the same ordering as the concrete syntax, i.e. `string loc list * pattern`.